src/AngleSharp.Js/AngleSharp.Js.csproj

```diff
@@ -1,28 +1,24 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <AssemblyName>AngleSharp.Js</AssemblyName>
-    <RootNamespace>AngleSharp.Js</RootNamespace>
-    <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netstandard2.0;net6.0;net7.0;net8.0</TargetFrameworks>
-    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">netstandard2.0;net462;net472;net6.0;net7.0;net8.0</TargetFrameworks>
-    <SignAssembly>true</SignAssembly>
```

Comment:
This is a breaking change - can you please leave such things untouched? Thanks!

Reply:
Which part? 

* Assembly name and namespace default to project's name (`AngleSharp.Js`) 
* Directory.build props gives the `<SignAssembly>true</SignAssembly>` and `<SignAssembly>true</SignAssembly>`
* targets are the same as before, defaults for all platforms and on windows the additional full framework ones

Reply:
(comment was about SignAssembly)

Yeah did not see it - only saw the removal. True its already part of the build props and thus should not be repeated. Thanks for clarifying.